apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/EndpointJsonResponseComparatorIT.java

```diff
@@ -164,7 +163,7 @@ public void verifyCorrectEndpointResponse() {
    * Generates the "golden" files, i.e. the approved responses to compare to. Run by commenting out
    * the <code>@Ignore</code> annotation and running this method as JUnit.
    */
-  @Ignore
+  // @Ignore
```

Comment:
Don't we leave the @Ignore active after generating the .json files?

Reply:
yes thannk u!!